src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -169,7 +173,9 @@ function PolicyAccountingPage({policy, connectionSyncProgress, isConnectionDataF
                 wrapperStyle: [styles.sectionMenuItemTopDescription],
                 shouldShowRightComponent: true,
                 title: integrationData?.title,
-
+                errorText: hasSynchronizationError(policy, connectedIntegration) ? translate('workspace.accounting.syncError', connectedIntegration) : undefined,
+                errorTextStyle: {marginTop: 8},
+                shouldShowErrorTextRedDot: true,
```

Comment:
should it be always true?

Reply:
In the majority of cases we use this error text without a red dot, so I decided to use shouldShowErrorTextRedDot this way.